src/app/interfaces/CustomProps.ts

```diff
@@ -0,0 +1,4 @@
+export interface CustomProps {
+    [str: string]: any;
+    [num: number]: any;
```

Comment:
why `number`?

Reply:
I assume to be able to have numeric keys. I took this interface from single-spa as is

Reply:
it's strange, but anyway it's not a problem, so let it be as is